keepkeylib/transport_webusb.py

```diff
@@ -77,8 +79,10 @@ def _close(self):
 
     @classmethod
     def enumerate(cls):
-        cls.context = usb1.USBContext()
-        cls.context.open() #TODO: are there any extra closing steps that need to be taken?
+        if not cls.context:
+            cls.context = usb1.USBContext()
+            cls.context.open()
+            atexit.register(cls.context.close)
```

Comment:
atexit needs to be imported... `import atexit` .. no?  Why is it this didn't crash? ;)